apple/internal/partials/swift_dylibs.bzl

```diff
@@ -65,7 +65,7 @@ File object that represents a directory containing the Swift dylibs to package f
 # OS version for the current target and platform is equal or above to the versions defined here,
 # then we can skip copying the Swift dylibs into Frameworks and SwiftSupport.
 _MIN_OS_PLATFORM_SWIFT_PRESENCE = {
-    "ios": apple_common.dotted_version("12.2"),
+    "ios": apple_common.dotted_version("15.2"),
```

Comment:
Need to check if it's 15.0 or 15.2, i think it's 15.0 but there were some issues so I'm not sure how that plays into this